packages/ods-react/src/components/tag/src/components/tag/Tag.tsx

```diff
@@ -0,0 +1,86 @@
+import classNames from 'classnames';
+import { type ComponentPropsWithRef, type FC, type JSX, type KeyboardEvent, type MouseEvent, forwardRef, useCallback } from 'react';
+import { Icon } from '../../../../icon/src';
+import { TAG_COLOR, type TagColor } from '../../constants/tag-color';
+import { TAG_SIZE, type TagSize } from '../../constants/tag-size';
+import style from './Tag.module.scss';
+
+interface TagRemoveEventDetail {
```

Comment:
not used, as you don't pass any id on the callback call, maybe can be remove totally?